opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/aggregation.py

```diff
@@ -58,25 +61,32 @@ class AggregationTemporality(IntEnum):
     CUMULATIVE = 2
 
 
-class _Aggregation(ABC, Generic[_PointVarT]):
+class _Aggregation(ABC, Generic[_DataPointVarT]):
     def __init__(self, attributes: Attributes):
         self._lock = Lock()
         self._attributes = attributes
+        self._previous_point = None
```

Comment:
My concern with this is that each aggregation subclass has to remember to set it's previous_point, which is error prone

Reply:
I think it is fine for the time being, only 2 classes have to deal with previous point right now.